fs/ops.go

```diff
@@ -181,14 +181,26 @@ func copyDirectory(source, dest string) error {
 			}
 			continue
 		}
-		// TODO: handle symlinks
-		if err := copyFile(sourcePath, destPath); err != nil {
+		if entry.Mode()&os.ModeSymlink != 0 {
+			if err := copySymLink(sourcePath, destPath); err != nil {
+				return err
+			}
+		} else if err := copyFile(sourcePath, destPath); err != nil {
```

Comment:
nit: I don't think we need the `else if`. It can `continue` in the success case. Alternatively it could use a switch/case.